api/src/main/java/org/apache/gravitino/credential/ADLSTokenCredential.java

```diff
@@ -0,0 +1,77 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.apache.gravitino.credential;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
+
+/** ADLS SAS token credential. */
+public class ADLSTokenCredential implements Credential {
+
+  /** ADLS SAS token credential type. */
+  public static final String ADLS_SAS_TOKEN_CREDENTIAL_TYPE = "adls-sas-token";
+  /** ADLS SAS token used to access ADLS data. */
+  public static final String GRAVITINO_ADLS_SAS_TOKEN = "adls-sas-token";
```

Comment:
I'd suggest we make them private.

Reply:
I change `ADLS_SAS_TOKEN_CREDENTIAL_TYPE ` to private.
But I need to use `GRAVITINO_ADLS_SAS_TOKEN ` in `CredentialPropertyUtils`

Reply:
This PR is ready for review now!